src/store/singing.ts

```diff
@@ -1265,25 +1268,17 @@ export const singingStore = createPartialStore<SingingStoreTypes>({
           });
         };
 
-        if (!filePath) {
-          filePath = await window.backend.showImportFileDialog({
-            title: "MIDI読み込み",
-            name: "MIDI",
-            extensions: ["mid", "midi"],
-          });
-          if (!filePath) return;
-        }
-
+        // NOTE: トラック選択のために一度ファイルを読み込んでいるので、Midiを渡すなどでもよさそう
+        // 現状このまま
```

Comment:
`現状このまま`が何を意図しているのかちょっと分かりづらいかもです。
消しちゃってもいいかも。
```suggestion
        // NOTE: トラック選択のために一度ファイルを読み込んでいるので、Midiを渡すなどでもよさそう
```